### PR TITLE
Adding new MPI tests to code coverage

### PR DESCRIPTION
**Pull request by john-science**
## What is the change?

Adding new MPI tests to code coverage

## Why is the change being made?

I want our new MPI tests to show up in the code coverage.

---

## Checklist

- [X] This PR has only [one purpose or idea](https://terrapower.github.io/armi/developer/tooling.html#one-idea-one-pr).
- [X] [Tests](https://terrapower.github.io/armi/developer/tooling.html#test-it) have been added/updated to verify that the new/changed code works.

- [X] The code style follows [good practices](https://terrapower.github.io/armi/developer/standards_and_practices.html).
- [X] The commit message(s) follow [good practices](https://terrapower.github.io/armi/developer/tooling.html).

- [X] The [release notes](https://terrapower.github.io/armi/release/index.html) (location `doc/release/0.X.rst`) are up-to-date with any important changes.
- [X] The [documentation](https://terrapower.github.io/armi/developer/tooling.html#document-it) is still up-to-date in the `doc` folder.
- [X] No [requirements](https://terrapower.github.io/armi/developer/tooling.html#watch-for-requirements) were altered.
- [X] The dependencies are still up-to-date in `pyproject.toml`.

### CODE REVIEW

File: armi/tests/test_mpiParameters.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Copyright 2019 TerraPower, LLC
```
Comment (keckler) [inline armi/tests/test_mpiParameters.py:1]: Copyright year should be 2023?

File: armi/tests/test_mpiParameters.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+from armi import context
+from armi.reactor import composites
+from armi.reactor import parameters
+
+# determine if this is a parallel run, and MPI is installed
+MPI_EXE = None
+if find_executable("mpiexec.exe") is not None:
+    MPI_EXE = "mpiexec.exe"
+elif find_executable("mpiexec") is not None:
+    MPI_EXE = "mpiexec"
```
Comment (keckler) [inline armi/tests/test_mpiParameters.py:27]: I think the corresponding block of code that is in the `test_parameters()` file can be removed, right?

**Review by keckler — COMMENTED** [review]
One small question for you.

File: armi/tests/test_mpiParameters.py (hunk 2 above)
Comment (john-science) [reply armi/tests/test_mpiParameters.py:27]: Fixed.  Good call.  Sorry about that.

**Review by keckler — APPROVED** [review]

**Pull request merged**